src/Interface/Controls/PluginCard.as

```diff
@@ -21,14 +21,59 @@ namespace Controls
 		}
 
 		// Draw an installed tag on top of the image if it's installed
+		uint tagRow = 0;
+		vec2 tagPos;
 		if (plugin.GetInstalledPlugin() !is null) {
-			vec2 tagPos = windowPos + imagePos + vec2(6, 6);
+			tagPos = windowPos + imagePos + vec2(6, 6 + (30*tagRow));
 			DrawTag(tagPos, Icons::CheckCircle + " Installed", Controls::TAG_COLOR_PRIMARY);
+			tagRow++;
 
 			// Draw an updatable tag on top of the image if it's installed and updatable
 			if (GetAvailableUpdate(plugin.m_siteID) !is null) {
-				tagPos = windowPos + imagePos + vec2(6, 36);
-				DrawTag(tagPos, Icons::ArrowCircleUp + " Update!", Controls::TAG_COLOR_WARNING);
+				tagPos = windowPos + imagePos + vec2(6, 6 + (30*tagRow));
+				string text = Icons::ArrowCircleUp + " Update!";
+				DrawTag(tagPos, text, Controls::TAG_COLOR_WARNING);
+				tagRow++;
+
+				// if updatable, show changelog in tooltip
+				if(Setting_ChangelogTooltips) {
+					// hacky invisible button to tooltip only on tag and not entire PluginCard
+					vec2 cursor = UI::GetCursorPos();
+					UI::SetCursorPos(tagPos - windowPos);
+					vec2 textSize = Draw::MeasureString(text);
+					vec2 tagSize = textSize + Controls::TAG_PADDING * 2;
+					UI::InvisibleButton(plugin.m_id + "_updatable", tagSize);
+					UI::SetCursorPos(cursor);
+
+					if (UI::IsItemHovered()) {
+						UI::SetNextWindowSize(400, -1, UI::Cond::Always);
+						UI::BeginTooltip();
+						PluginChangelog(plugin, true);
+						UI::EndTooltip();
+					}
```

Comment:
Can these use `DrawTag` too?

Reply:
I refactored the invis button code to a new function, I still had to leave the IsItemHovered() if block out since PluginChangelog() does a lot of formatting stuff and the other is just a UI::Text(). But at least it's still less duplicated code!